### ISSUE TITLE
Database not used when writing in Ne04jWriter 

### ISSUE THREAD

**Issue by asmita-mukherjee** [neo4j/neo4j-graphrag-python#214]
Code Link : https://github.com/neo4j/neo4j-graphrag-python/blob/8285d56e53af36d3a0a5dadd094e0982a128e6fd/src/neo4j_graphrag/experimental/components/kg_writer.py#L160

The database initialized should also be the database where the graph is written, however it currently uses the default database.

**Comment by stellasia** [comment]
Hi @asmita-mukherjee , 

Thank you for the report, you're right, this is a bug, we're working on it.

In the meantime, maybe you can specify the database when you initialize the driver:
```
import neo4j
driver = neo4j.GraphDatabase.driver(URI, auth=AUTH, database=DATABASE)
```
in which case, you do not have to specify it in the writer.

**Comment by asmita-mukherjee** [comment]
sure thanks :)

**Comment by stellasia** [comment]
Hi @asmita-mukherjee ,

This was fixed in [1.2.1](https://github.com/neo4j/neo4j-graphrag-python/releases/tag/1.2.1).
